code/addons/test/src/components/GlobalErrorModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import { Button, IconButton, Modal } from 'storybook/internal/components';
+import { useStorybookApi } from 'storybook/internal/manager-api';
+
+import { CloseIcon, SyncIcon } from '@storybook/icons';
+import { styled } from '@storybook/theming';
+
+import { TROUBLESHOOTING_LINK } from '../constants';
+
+interface GlobalErrorModalProps {
+  error: string;
+  open: boolean;
+  onClose: () => void;
+  onRerun: () => void;
+}
+
+const ModalBar = styled.div({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  padding: '6px 6px 6px 20px',
+});
+
+const ModalActionBar = styled.div({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+});
+
+const ModalTitle = styled.div(({ theme: { typography } }) => ({
+  fontSize: typography.size.s2,
+  fontWeight: typography.weight.bold,
+}));
+
+const ModalStackTrace = styled.pre(({ theme }) => ({
+  whiteSpace: 'pre-wrap',
+  overflow: 'auto',
+  maxHeight: '60vh',
+  margin: 0,
+  padding: `20px`,
+  fontFamily: theme.typography.fonts.mono,
+  fontSize: '12px',
+  borderTop: `1px solid ${theme.appBorderColor}`,
```

Comment:
seems like the border is round?
<img width="325" alt="image" src="https://github.com/user-attachments/assets/2825cbe2-43c3-44b2-ab92-3c1cb1f7ea94">

Reply:
Thanks!